modules/29_CES_parameters/calibrate/preloop.gms

```diff
@@ -8,17 +8,15 @@
 $OFForder
 
 
-*** ==================================================================================
 *** ==================================================================================
 ***
-*** FOR A DETAILED DOCUMENTATION OF THIS FILE, SEE THE GOXYGEN DOCUMENTATION, CHAPTER
+*** For a detailed documentation of this file, see the goxygen documentation, chapter
```

Comment:
Which begs the question: why is the documentation in `realization.gms` and not with the code in `preloop.gms`?

Reply:
I don't mind moving it to preloop, though the first part of the documentation is kind of for the whole module, and tearing it in half would also be bad.

Reply:
Well, we can rearrange when we refactor the realisation.